src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm2/PlanTypeOptions/PlanTypeOptions.tsx

```diff
@@ -25,58 +26,73 @@ export type NewPlanType =
   | 'users-teamy'
 
 interface PlanTypeOptionsProps {
-  setSelectedPlan: (x: z.infer<typeof IndividualPlanSchema>) => void
+  multipleTiers: boolean
   setFormValue: (x: string, y: string) => void
+  setSelectedPlan: (x: z.infer<typeof IndividualPlanSchema>) => void
 }
 
 const PlanTypeOptions: React.FC<PlanTypeOptionsProps> = ({
+  multipleTiers,
   setFormValue,
   setSelectedPlan,
 }) => {
   const { provider, owner } = useParams<{ provider: string; owner: string }>()
   const { data: plans } = useAvailablePlans({ provider, owner })
   const { data: accountDetails } = useAccountDetails({ provider, owner })
   const { proPlanYear } = useProPlans({ plans })
+
   const { sentryPlanYear } = findSentryPlans({ plans })
   const { teamPlanYear } = findTeamPlans({
     plans,
   })
-  const [option, setOption] = useState<'Pro' | 'Team'>('Pro')
-
+  const hasTeamPlans = shouldDisplayTeamCard({ plans })
   const plan = accountDetails?.rootOrganization?.plan ?? accountDetails?.plan
   const isSentryUpgrade = canApplySentryUpgrade({ plan, plans })
   const yearlyProPlan = isSentryUpgrade ? sentryPlanYear : proPlanYear
 
-  return (
-    <div className="flex w-fit flex-col gap-2">
-      <h3 className="font-semibold">Plan</h3>
-      <div className="inline-flex items-center gap-2">
-        <OptionButton
-          type="button"
-          active={option}
-          onChange={({ text }) => {
-            if (text === 'Pro') {
-              setSelectedPlan(yearlyProPlan)
-              setFormValue('newPlan', yearlyProPlan?.value)
-            } else {
-              setSelectedPlan(teamPlanYear)
-              setFormValue('newPlan', teamPlanYear?.value)
-            }
-            setOption(text)
-          }}
-          options={[
-            {
-              text: 'Pro',
-            },
-            {
-              text: 'Team',
-            },
-          ]}
-        />
-        {option === 'Team' && <p>Up to {TEAM_PLAN_MAX_ACTIVE_USERS} users</p>}
+  const [option, setOption] = useState<'Pro' | 'Team'>('Pro')
```

Comment:
Sorry again just seeing this now, may also be worth extracting this guy here to an object